### PR TITLE
Wip/trending section

### PR DESCRIPTION
**Pull request by AndzelikaB**

### CODE REVIEW

File: src/app/components/home/trending-section/trending-section.component.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  styleUrls: ['./trending-section.component.scss'],
+})
+export class TrendingSectionComponent {
+  trendingMovies: any;
+  theatreMovies: any;
+  popularMovies: any;
+
+  constructor(private http: HttpClient) {}
+
+  listBook() {
```
Comment (adrianGaluszka) [inline src/app/components/home/trending-section/trending-section.component.ts:16]: set returned type

File: src/app/components/home/trending-section/trending-section.component.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+@Component({
+  selector: 'app-trending-section',
+  templateUrl: './trending-section.component.html',
+  styleUrls: ['./trending-section.component.scss'],
+})
+export class TrendingSectionComponent {
+  trendingMovies: any;
+  theatreMovies: any;
+  popularMovies: any;
```
Comment (adrianGaluszka) [inline src/app/components/home/trending-section/trending-section.component.ts:12]: try avoid 'any' type for variables

File: src/app/components/home/trending-section/trending-section.component.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+  constructor(private http: HttpClient) {}
+
+  listBook() {
+    this.http
+      .get<any>('http://localhost:4200/assets/data/trending-movies.json')
```
Comment (adrianGaluszka) [inline src/app/components/home/trending-section/trending-section.component.ts:18]: type

File: src/app/components/initial-page/signup/signup.component.ts [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  signUp(): void {
+    this.authService.createUser(
+      this.email.value,
+      this.username.value,
+      this.password.value
+    );
+
+    this.username.reset();
+    this.email.reset();
+    this.password.reset();
```
Comment (adrianGaluszka) [inline src/app/components/initial-page/signup/signup.component.ts:40]: is it necessary if in line 41 is used method resetForm?

File: src/app/services/auth.service.ts [hunk 5]
```diff
@@ -7,24 +7,27 @@ import { HttpClient } from '@angular/common/http';
 })
 export class AuthService {
   wrongData: boolean = false;
+  signSuccess: boolean = false;
 
   get userIsLogged(): boolean {
-    const data: any = localStorage.getItem('userIsLogged');
-    const userIsLogged: boolean = JSON.parse(data);
-    return data ? userIsLogged : false;
+    const localStorageData: any = localStorage.getItem('userIsLogged');
```
Comment (adrianGaluszka) [inline src/app/services/auth.service.ts:13]: change any to string

File: src/app/services/auth.service.ts [hunk 6]
```diff
@@ -7,24 +7,27 @@ import { HttpClient } from '@angular/common/http';
...
+        const userLogged: boolean = res.find((db: any) => {
+          return (
+            (db.username === name || db.email === name) && db.password === pass
+          );
         });
-        if (user) {
-          localStorage.setItem('userIsLogged', JSON.stringify(true));
 
+        if (userLogged) {
+          localStorage.setItem('userIsLogged', JSON.stringify(true));
```
Comment (adrianGaluszka) [inline src/app/services/auth.service.ts:30]: Instead of 'JSON.stringify(true)' just try true

File: src/app/services/auth.service.ts [hunk 7]
```diff
@@ -36,8 +39,38 @@ export class AuthService {
     });
   }
 
+  createUser(name: any, email: any, pass: any) {
```
Comment (adrianGaluszka) [inline src/app/services/auth.service.ts:42]: set params types and returned type

File: src/app/services/auth.service.ts (hunk 6 above)
Comment (AndzelikaB) [reply src/app/services/auth.service.ts:30]: Argument of type 'boolean' is not assignable to parameter of type 'string'

**Pull request merged**